jenkins/internal-ci/generic/branch/cortx-ha.groovy

```diff
@@ -129,8 +129,8 @@ EOF
                 script {
                     def releaseBuild = build job: 'Release', propagate: true
                     env.release_build = releaseBuild.number
-                    env.release_build_location = "http://cortx-storage.colo.seagate.com/releases/cortx/github/$branch/$os_version/" +releaseBuild.number
```

Comment:
Please test this with one of the component job execution.

Reply:
Job : https://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/Subha_WS/job/cortx-ha-internal-ci/6/console